backend/edna.cabal

```diff
@@ -82,13 +82,15 @@ library
     bytestring,
     containers,
     data-default,
+    filepath,
     fmt,
     microlens,
     microlens-platform,
     network-uri,
     network-uri-json,
     optparse-applicative,
     postgresql-simple,
+    process,
```

Comment:
I saw in Slack that cool boyz use `typed-process`, did you consider using it?

Reply:
Yes, but only if we will have time. So, if I will rebase and implement all other fixes before demo and we will have time to use `typed-process` I will add it